### PR TITLE
Add email checking teams are still wanted

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add an email checking institutions still want the same number of teams they started off with.

This covers both institutions with 1 team which is inactive, _and_ those with multiple teams where one / many are inactive.

### CODE REVIEW

File: SR2021/2021-01-12-team-numbers.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+subject: Student Robotics Teams
+---
+
+Hi,
+
+If you're reading this email, it means you have at least 1 team from your institution who seem inactive on Discord, or who have few members.
+
+If you have teams requiring further guidance and support, the Discord is the best place to get support. Filled with knowledgable volunteers, competitors and team leaders, all trying to help.
+
+If you no longer wish to have as many teams as you registered for, please let us know. Dropping out a team and merging it and merging it with an existing one becomes more unfair the further into the competition we get.
```
Comment (antoinepetty) [inline SR2021/2021-01-12-team-numbers.md:12]:
```suggestion
If you no longer wish to have as many teams as you registered for, please let us know. Dropping out a team and merging it with an existing one becomes more unfair the further into the competition we get.
```

File: SR2021/2021-01-12-team-numbers.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+to: SR2021 institutions with inactive teams
+subject: Student Robotics Teams
+---
+
+Hi,
+
+If you're reading this email, it means you have at least 1 team from your institution who seem inactive on Discord, or who have few members.
```
Comment (PeterJCLaw) [inline SR2021/2021-01-12-team-numbers.md:8]: Presumably this is actually only going to institutions that have multiple teams assigned? I suggest that we probably want a slightly different email if they only have the one team and aren't active?

File: SR2021/2021-01-12-team-numbers.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+If you have teams requiring further guidance and support, the Discord is the best place to get support. Filled with knowledgable volunteers, competitors and team leaders, all trying to help.
```
Comment (PeterJCLaw) [inline SR2021/2021-01-12-team-numbers.md:10]: Perhaps "available to help"? "trying" almost implies they're failing.
```suggestion
If you have teams requiring further guidance and support, the Discord is the best place to get support. Filled with knowledgable volunteers, competitors and team leaders, all available to help.
```

Maaybe also mention that they can email us if they're stuck but can't use Discord?

**Review by PeterJCLaw — APPROVED** [review]
:+1: some suggestions for further improvement

File: SR2021/2021-01-12-team-numbers.md (hunk 2 above)
Comment (RealOrangeOne) [reply SR2021/2021-01-12-team-numbers.md:8]: Yeah this email is more about culling institutions with too many teams. Inactivity is definitely better measured after the league.

Comment (PeterJCLaw) [reply SR2021/2021-01-12-team-numbers.md:8]: Maybe up the the To then?

Comment (PeterJCLaw) [inline SR2021/2021-01-12-team-numbers.md:8]: I wonder if a slightly friendlier tone might help here? I just re-read this and it sounds very direct.

Perhaps something like this is a bit kinder?
```suggestion
We noticed that one or more of your teams doesn't seem active on Discord, or has very few members. We want to ensure that every team has the opportunity to shine and gets the support they need.
```

Comment (RealOrangeOne) [reply SR2021/2021-01-12-team-numbers.md]: 100% agree! :tada:

**Review by PeterJCLaw — APPROVED** [review]

File: SR2021/2021-01-12-team-numbers.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+to: SR2021 institutions with inactive teams
+subject: Student Robotics Teams
+---
+
+Hi,
+
+We noticed that one or more of your teams doesn't seem active on Discord, or has very few members. We want to ensure that every team has the opportunity to shine and gets the support they need.
+
+If you have teams requiring further guidance and support, the Discord is the best place to get support. Filled with knowledgable volunteers, competitors and team leaders, all available to help. Whilst most of the discussions and support will be happening here, if you are unable to or do not wish to create an account with Discord you can still [contact us](mailto:teams@studentrobotics.org) for support, however the response times will be longer.
```
Comment (WillB97) [inline SR2021/2021-01-12-team-numbers.md:10]:
```suggestion
If you have teams requiring further guidance and support, the Discord is the best place to get support. Filled with knowledgeable volunteers, competitors and team leaders, all available to help. Whilst most of the discussions and support will be happening here, if you are unable to or do not wish to create an account with Discord you can still [contact us](mailto:teams@studentrobotics.org) for support, however the response times will be longer.
```
nit: spelling

**Pull request merged**